src/runtime_src/core/common/info_aie.cpp

```diff
@@ -70,10 +70,25 @@ addnodelist(const std::string& search_str, const std::string& node_str,
 static void
 populate_aie_dma(const boost::property_tree::ptree& pt, boost::property_tree::ptree& pt_dma)
 {
+  boost::property_tree::ptree fifo_pt;
   boost::property_tree::ptree mm2s_array;
   boost::property_tree::ptree s2mm_array;
   boost::property_tree::ptree empty_pt;
 
+  // Extract FIFO COUNT information
+  // Sample sysfs entry for DMA
+  // cat /sys/devices/platform/<aie_path>/aiepart_0_50/47_0/dma
+  // ichannel_status: mm2s: idle|idle, s2mm: idle|idle
+  // queue_size: mm2s: 0|0, s2mm: 0|0
+  // queue_status: mm2s: okay|okay, s2mm: okay|okay
+  // current_bd: mm2s: 0|0, s2mm: 0|0
+  // fifo_len: 0|0
+  auto fifo_info = pt.get_child("dma.fifo_len", empty_pt).begin();
+  fifo_pt.put("fifo_counter0", fifo_info->second.data());
```

Comment:
This code assumes that there are only 2 counters,  can there be 1, 3 or more counters?

For scaling purposes, should there be an array of "fifo_counters" instead of adding them to a dictionary.

Note
Bad things happen if there are 0,1, 3, 4+ counters return.  The code should be more resilient if this should occur.

Reply:
By looking at following spec
http://cervino-doc/r2p19/tile_links/xregdb_me_tile_doc.html#tile_memory___DMA_FIFO_Counter
You can safely assume that, there are two fifo counters.

Reply:
Lets chat for the code should be more abstract especially since the AIE IPs are being actively updated and this code will need to support version 1, 2, and possibly 3.

So while the IP registers are harden at the moment, forcing the names to have the index value as well is not good.

Reply:
I have made the changes. Please review it back. Thanks